x-pack/plugins/maps/public/layers/sources/mvt_vector_source/mvt_single_layer_vector_source.ts

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import uuid from 'uuid/v4';
+import { ImmutableSourceProperty } from '../source';
+import { SingleTiledVectorLayer } from '../../tiled_vector_layer';
+import {
+  AbstractVectorSource,
+  GeoJsonWithMeta,
+  ITiledSingleLayerVectorSource,
+} from '../vector_source';
+import { MAX_ZOOM, MIN_ZOOM, SOURCE_TYPES } from '../../../../common/constants';
+import { VECTOR_SHAPE_TYPES } from '../vector_feature_types';
+import { IField } from '../../fields/field';
+import { registerSource } from '../source_registry';
+import { getDataSourceLabel, getUrlLabel } from '../../../../common/i18n_getters';
+import {
+  LayerDescriptor,
+  MapExtent,
+  TiledSingleLayerVectorSourceDescriptor,
+  VectorSourceRequestMeta,
+  VectorSourceSyncMeta,
+} from '../../../../common/descriptor_types';
+import { VectorLayerArguments } from '../../vector_layer';
+
+export const sourceTitle = i18n.translate('xpack.maps.source.ems_xyzVectorTitle', {
```

Comment:
Change translation id from `xpack.maps.source.ems_xyzVectorTitle`. This is not EMS nor xyz source